### PR TITLE
{afd} Fd api version update to 2024

### PR DESCRIPTION
**Pull request by Ptnan7**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -936,6 +936,12 @@
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
+# ServiceLabel: %Network - Front Door %Mgmt
+# ServiceOwners:                                                   @cdnfdsuppgithub
```
Comment (jsquire) [inline .github/CODEOWNERS:940]: The account `cdnfdsuppgithub` does not have the needed public organizations nor write permissions to the repository.   This needs to be resolved before it can be added as a code owner.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -936,6 +936,9 @@
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
+# PRLabel: %Network - Front Door
```
Comment (jsquire) [inline .github/CODEOWNERS:939]: This isn't correct.   Please refer to my earlier comment for what form this entry needs to take.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -945,6 +945,12 @@
 # PRLabel: %Large Instance
 /sdk/azurelargeinstance/Azure.ResourceManager.*/                   @8Gitbrix @archerzz @ArcturusZhang @ArthurMa1978
 
+# PRLabel: %Network - Front Door
+/sdk/frontdoor/                                                    @ptnan7
```
Comment (jsquire) [inline .github/CODEOWNERS:949]: This should be scoped to the Resource Manager package not the entire slug.

Comment (jsquire) [inline .github/CODEOWNERS:949]:
```suggestion
/sdk/frontdoor/Azure.ResourceManager.*/                            @ptnan7
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -945,6 +945,12 @@
 # PRLabel: %Large Instance
 /sdk/azurelargeinstance/Azure.ResourceManager.*/                   @8Gitbrix @archerzz @ArcturusZhang @ArthurMa1978
 
+# PRLabel: %Network - Front Door
```
Comment (jsquire) [inline .github/CODEOWNERS:948]: Why is this sitting in the middle of the "Large Instance" block?   Please move down so that both "Large Instance" sections are together.

**Review by jsquire — DISMISSED** [review]

File: sdk/frontdoor/Azure.ResourceManager.FrontDoor/CHANGELOG.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
```
Comment (live1206) [inline sdk/frontdoor/Azure.ResourceManager.FrontDoor/CHANGELOG.md:3]: update the version in csproj file as well

**Review by live1206 — COMMENTED** [review]
Merge from latest main to resolve the conflicts

File: sdk/frontdoor/Azure.ResourceManager.FrontDoor/src/Generated/Models/GroupByVariable.cs [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.FrontDoor.Models
+{
+    /// <summary> Describes the variables available to group the rate limit requests. </summary>
+    public partial class GroupByVariable
```
Comment (live1206) [inline sdk/frontdoor/Azure.ResourceManager.FrontDoor/src/Generated/Models/GroupByVariable.cs:14]: The name is too general, please rename it to be more specific to avoid conflicts with other RPs, such as prepend PR name. You can refer to `prepend-rp-prefix` in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

File: sdk/frontdoor/Azure.ResourceManager.FrontDoor/src/Generated/Models/VariableName.cs [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.FrontDoor.Models
+{
+    /// <summary> Describes the supported variable for group by. </summary>
+    public readonly partial struct VariableName : IEquatable<VariableName>
```
Comment (live1206) [inline sdk/frontdoor/Azure.ResourceManager.FrontDoor/src/Generated/Models/VariableName.cs:14]: The name is too general, please rename it to be more specific to avoid conflicts with other RPs, such as prepend PR name. You can refer to `prepend-rp-prefix` in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

File: sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md [hunk 8]
```diff
@@ -20,7 +20,6 @@
 ## 1.2.0 (2023-11-21)
 
 ### Features Added
-
```
Comment (live1206) [inline sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md:23]: revert this change

File: sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs [hunk 9]
```diff
@@ -125,6 +125,8 @@ public static partial class FrontDoorExtensions
...
         public static Azure.AsyncPageable<Azure.ResourceManager.FrontDoor.FrontDoorResource> GetFrontDoorsAsync(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static Azure.ResourceManager.FrontDoor.FrontDoorWebApplicationFirewallPolicyCollection GetFrontDoorWebApplicationFirewallPolicies(this Azure.ResourceManager.Resources.ResourceGroupResource resourceGroupResource) { throw null; }
+        public static Azure.Pageable<Azure.ResourceManager.FrontDoor.FrontDoorWebApplicationFirewallPolicyResource> GetFrontDoorWebApplicationFirewallPoliciesByFrontDoorWebApplicationFirewallPolicy(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (live1206) [inline sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs:128]: tests are missing for the newly added operations since this is GA version

**Review by live1206 — CHANGES_REQUESTED** [review]
Tests are missing for GA release

**Review by live1206 — APPROVED** [review]
